web-console/src/components/table-column-selector/__snapshots__/table-column-selector.spec.tsx.snap

```diff
@@ -14,7 +14,12 @@ exports[`table column matches snapshot 1`] = `
       <span
         class="bp3-button-text"
       >
-        Columns
+        Columns 
+        <span
+          class="counter"
+        >
+          (0/3)
```

Comment:
May be worth it for the test to validate a scenario where some of the columns are shown